components/sup/src/manager/mod.rs

```diff
@@ -95,6 +98,19 @@ const PROC_LOCK_FILE: &'static str = "LOCK";
 
 static LOGKEY: &'static str = "MR";
 
+/// Captures the different operations that can be performed on a
+/// service.
```

Comment:
I like a good comment, but with the exception of `Restart`, I think these aren't adding anything that isn't communicated by the thoughtful names.